content/en/security/application_security/setup/java/linux.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Setup App and API Protection for Java on Linux
+code_lang: linux
+type: multi-code-lang
+code_lang_weight: 30
+further_reading:
+- link: "/security/application_security/how-it-works/"
+  tag: "Documentation"
+  text: "How App and API Protection Works"
+- link: "/security/default_rules/?category=cat-application-security"
+  tag: "Documentation"
+  text: "OOTB App and API Protection Rules"
+- link: "/security/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting App and API Protection"
+---
+
+{{< partial name="app_and_api_protection/callout.html" >}}
+
+{{< partial name="app_and_api_protection/java/overview.html" >}}
+
+This guide explains how to set up App and API Protection (AAP) for Java applications running on Linux. The setup involves:
+1. Installing the Datadog Agent
+2. Configuring your Java application
+3. Enabling AAP monitoring
+
+## Prerequisites
+
+- Linux operating system
+- Java application
+- Root or sudo privileges
+- Systemd (for service management)
+
+## Setup
+
+### 1. Install the Datadog Agent
+
+#### Using the one-line install script (recommended)
+
+```bash
+DD_AGENT_MAJOR_VERSION=7 DD_API_KEY=<YOUR_API_KEY> DD_SITE=<YOUR_DD_SITE> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+```
+
+#### Manual installation
+
+For Debian/Ubuntu:
+```bash
+sudo apt-get update
+sudo apt-get install -y apt-transport-https curl gnupg
+sudo sh -c "echo 'deb https://apt.datadoghq.com/ stable 7' > /etc/apt/sources.list.d/datadog.list"
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A2923DFF56EDA6E76E55E492D3A80E30382E94DE
+sudo apt-get update
+sudo apt-get install -y datadog-agent
+```
+
+For RHEL/CentOS:
+```bash
+sudo yum install -y https://yum.datadoghq.com/rpm/x86_64/datadog-agent-7.x.x-1.x86_64.rpm
+```
+
+## Library setup
+
+To enable AAP capabilities, you need the Datadog Java tracing library (version 0.94.0 or higher) installed in your application environment.
+
+### Download the library
+
+Download the latest version of the Datadog Java library:
+
+```bash
+wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
+```
+
+### Verify compatibility
+
+To check that your service's language and framework versions are supported for AAP capabilities, see [Single Step Instrumentation Compatibility][2].
+
+## Service configuration
+
+### Run your application with AAP enabled
+{{% tabs %}}
```

Comment:
I rather think for your Java case that tabs apply to different configuration styles like env var vs jar CLI options.
In each tab, state the fact APM tracing can be disabled. Also, don't call it standalone billing as this was an internal naming about the implementation.

Other than that, it's not necessarily simple to think of a clean solution exposing both deployment options (with or without APM tracing). We also need to add a shortcut anchor link at the beginning of this page for APM-enabled services to shortcut the setup instruction straight to "redploy with DD_APPSEC_ENABLED=true".